main.go

```diff
@@ -112,20 +104,23 @@ func main() {
 		os.Exit(1)
 	}
 
-	if err := initHostedClusters(mgr); err != nil {
-		setupLog.Error(err, "Init hosted clusters")
-	}
+	//Adding HyperShiftLogForwarder controller
+	_, err = hypershiftlogforwarder.NewHyperShiftLogForwarderReconciler(mgr)
```

Comment:
Do we need register when start the main manager?

Reply:
Yes, Still need to register the controller with the main manager. since HLF controller initially creates sub managers for existing hosted clusters but it proceeds via `go routine` and may not block main-manager start.